### PR TITLE
TAJO-1977: Cannot recognize the space-contained tablename and databasename

### PR DESCRIPTION
**Pull request by dkhwangbo**
I found some bugs in using ```\c``` and ```\d```. Purpose of this PR is to fixing them.

### CODE REVIEW

**Comment by jihoonson** [comment]
You need to add some tests.

**Comment by dkhwangbo** [comment]
May I use used unit? or create another new unit? I guess I can make some test related of this PR with used unit like TestTajoCli::testConnectDatabase and TestTajoCli::testDescTable to add some code line more in that.

**Comment by jihoonson** [comment]
It will be better to add a new test.

**Comment by dkhwangbo** [comment]
@jihoonson I add a new unit test. Please review this PR.

File: tajo-core-tests/src/test/java/org/apache/tajo/cli/tsql/TestTajoCli.java [hunk 1]
```diff
@@ -228,17 +237,26 @@ private void verifyDescTable(String sql, String tableName, String resultFileName
...
     if (cluster.isHiveCatalogStoreRunning()) {
-      tableName = "TEST_DESC_TABLE".toLowerCase();
+      tableName = testTableName.toLowerCase();
     } else {
-      tableName = "TEST_DESC_TABLE";
+      tableName = testTableName;
     }
 
     String sql = "create table \"" + tableName + "\" (col1 int4, col2 int4);";
-    verifyDescTable(sql, tableName, "testDescTable.result");
```
Comment (jihoonson) [inline tajo-core-tests/src/test/java/org/apache/tajo/cli/tsql/TestTajoCli.java]: If this file does not used anymore, please remove it.

**Comment by dkhwangbo** [comment]
@jihoonson  Thanks for your review. I remove ```testDescTable.result``` via ```git rm``` .

**Comment by jihoonson** [comment]
+1 LGTM.

**Pull request closed without merging**